app/models/user.rb

```diff
@@ -1,15 +1,12 @@
 class User < ActiveRecord::Base
   include TokenAuthenticatable
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  # :token_authenticatable, :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable,
          :recoverable, :rememberable,
-         :trackable, :validatable,
-         :token_authenticatable
+         :trackable, :validatable
   has_many :orders
   validates :name, presence: true
-  validates :email, presence: true, uniqueness: true
-  validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i
```

Comment:
Why did you remove this format validation? We should still have something like that in place.